src/containers/DownloadCsv/index.jsx

```diff
@@ -0,0 +1,19 @@
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+
+import fetchCsv from '../../data/actions/courseEnrollments';
+import DownloadCsv from '../../components/DownloadCsv';
+
+const mapStateToProps = state => ({
+  enterpriseId: state.portalConfiguration.enterpriseId,
+  csvLoading: state.courseEnrollments.csvLoading,
+  csvError: state.courseEnrollments.csvError,
+});
+
+const mapDispatchToProps = dispatch => ({
+  fetchCsvForMethod: (enterpriseId, csvFetchMethod) => {
```

Comment:
The `mapDispatchToProps` for this container should have a similar implementation to [how it's done for the table](https://github.com/edx/edx-portal/blob/8b672d5975aeeb94237cf1ea7f46f6bba4fef4c9/src/containers/TableContainer/index.jsx#L18). 

We should be passing `ownProps.id` and `ownProps.fetchMethod` into the `fetchCsv` method.

nit: Rename `fetchCsvForMethod` to `fetchCsv`.